src/SelectMenu/SelectMenuStyles.js

```diff
@@ -0,0 +1,287 @@
+import {keyframes, css} from 'styled-components'
+import {get} from '../constants'
+
+const animateModal = keyframes`
+  0% {
+    opacity: 0;
+    transform: scale(0.9);
+  }
+`
+export const wrapperStyles = css`
+  &[open] > summary::before {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 80;
+    display: block;
+    cursor: default;
+    content: ' ';
+    background: transparent;
+  }
+  // Remove marker added by the display: list-item browser default
+  > summary {
+    list-style: none;
+  }
+  // Remove marker added by details polyfill
+  > summary::before {
+    display: none;
+  }
+  // Remove marker added by Chrome
+  > summary::-webkit-details-marker {
+    display: none;
+  }
+`
+
+export const modalStyles = css`
+  position: relative;
+  z-index: 99; // Needs to be higher than .details-overlay's z-index: 80.
+  display: flex;
+  ${props => (props.filter ? 'height: 80%' : '')};
```

Comment:
Does this do anything? I don't see `filter` on the TS

Reply:
nope! removing it